test/get.js

```diff
@@ -10,3 +10,26 @@ test('get returns error when nothing found', function (t) {
     t.same(node.length, 0)
   })
 })
+
+test('getByVersion', function (t) {
+  var node = {
+    type: 'node',
+    changeset: '9',
+    lat: '-11',
+    lon: '-10',
+    timestamp: '2017-10-10T19:55:08.570Z'
+  }
+
+  t.plan(4)
+
+  var db = createDb()
+
+  db.create(node, function (err, elm) {
+    t.error(err, 'create worked')
+    t.equals(elm.timestamp, node.timestamp, 'timestamp preserved')
```

Comment:
Yeah I'm not sure timestamp should be preserved, created_at should be preserved though.